notebooks/03-query/05-lookup-essentials.ipynb

```diff
@@ -0,0 +1,1560 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ea96371b-d628-40c5-9ba8-14c094e16c20",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "SELECT\n",
+    "  \"TABLE_NAME\"\n",
+    "FROM \"INFORMATION_SCHEMA\".\"TABLES\"\n",
+    "WHERE \"TABLE_SCHEMA\" = 'lookup'\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7518d6e2-4c24-4ced-bcbf-7982877d3b83",
+   "metadata": {},
+   "source": [
+    "Run the following cell to see how the [`LOOKUP`](https://druid.apache.org/docs/27.0.0/querying/sql-functions#lookup) SQL function can also be used as an alternative to  `JOIN`, in the case of this example, could have been more complicated."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a21f16d7-c37c-4b74-b839-0b96721ce5b7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "SELECT\n",
+    "    LOOKUP(\"Origin\", \\'''' + lookup_name + '''\\') AS \"originAirportSize\",\n",
+    "    LOOKUP(\"Dest\", \\'''' + lookup_name + '''\\') AS \"destinationAirportSize\",\n",
+    "    COUNT(*) AS \"flights\",\n",
+    "    SUM(Distance) AS \"totalDistance\"\n",
+    "FROM \"example-flights-lookup\"\n",
+    "WHERE TIME_IN_INTERVAL(__time,'2005-11-30T11:00:00/2015-11-30T08:00:00')\n",
+    "GROUP BY 1, 2\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ffea65dc-00d5-4b33-87c3-6b2394a9bb38",
+   "metadata": {},
+   "source": [
+    "## Clean up"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2275a54-a4f0-42a5-b010-34bf99ae9d4a",
+   "metadata": {},
+   "source": [
+    "Run the following cell to drop the example table and to call the lookup API to delete the two lookups that you created."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1bbd14b5-d659-460f-bf9d-5059d44ff0e6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "druid.datasources.drop(\"example-flights-lookup\")\n",
+    "x = requests.delete(druid_host + '/druid/coordinator/v1/lookups/config/'+lookup_tier+'/'+lookup_name)\n",
+    "x = requests.delete(druid_host + '/druid/coordinator/v1/lookups/config/'+lookup2_tier+'/'+lookup2_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "54b8d5fe-ba85-4b5b-9669-0dd47dfbccd1",
+   "metadata": {},
+   "source": [
+    "## Summary\n",
+    "\n",
+    "* Lookups are key/value tables that exist in the `lookup` schema\n",
+    "* Druid surfaces APIs to allow for the management of Lookups, including their definition and deletion\n",
+    "* Data in lookups can be updated\n",
+    "* Lookup tables can be used in `JOIN` operations and via the `LOOKUP` functions\n",
+    "\n",
+    "## Learn more\n",
+    "\n",
+    "* Try using the `lookup` JSON-based ingestion function to `JOIN` to a Lookup schema table at ingestion time"
```

Comment:
Link to the docs here also for easy navigation

Reply:
Done.